libc/src/__support/macros/properties/types.h

```diff
@@ -17,6 +17,8 @@
 #include "src/__support/macros/properties/cpu_features.h"
 #include "src/__support/macros/properties/os.h"
 
+#include <stdint.h> // __SIZEOF_INT128__
```

Comment:
Can you include `include/llvm-libc-macros/stdint-macros.h` instead?

Reply:
Not in this PR for the same reasons as https://github.com/llvm/llvm-project/pull/83441#discussion_r1517501301